tests/conftest.py

```diff
@@ -0,0 +1,13 @@
+import os
+import tempfile
+
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def tmp_dataset_dir():
+    """Generate a temporary directory for Minari datasets."""
+    tmp_dir = tempfile.TemporaryDirectory()
+    os.environ["MINARI_DATASETS_PATH"] = tmp_dir.name
```

Comment:
should you set `MINARI_DATASETS_PATH` back to the original value after it?

I guess if you interrupt tests for some reasons, there is a risk this will remain written anyway, but c'est la vie, I don't see a better way to do it

Reply:
Good idea, thanks! I have rewritten it to use MonkeyPatch, which automatically undoes the environment variable change upon exiting the with block.